README.md

```diff
@@ -1,15 +1,48 @@
 # WebKitRequirements
 > Third party packages required for building the open source WebKit port for Windows.
 
+## Setup
+
+WebKitRequirements uses [vcpkg](https://github.com/microsoft/vcpkg) to drive
+building the libraries. A helper script, `Install-Vcpkg.ps1`, configures
+`vcpkg` to be able to build the repository.
+
+```powershell
+> Install-Vcpkg.ps1 -vcpkgPath <path-to-vcpkg>

```

Comment:
Maybe add `[-update]` here and then the following can serve as a description for that case?